types.go

```diff
@@ -912,6 +914,12 @@ type Runtime struct {
 	Description      string `json:"description"`
 }
 
+type StaticIP struct {
+	Value  string
```

Comment:
how about making this a net.IP ?

Reply:
not sure. this would be mostly just nice but not needed since it will end up as strings anyways in the tf provider.

Reply:
true, but there are people using the go-sdk directly and I like it when SDKs come with good devex.

Reply:
done